packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -310,25 +343,50 @@ export class Stepper implements LocalizedComponent, T9nComponent {
     updateMessages(this, this.effectiveLocale);
   }
 
-  private itemMap = new Map<HTMLCalciteStepperItemElement, { position: number; content: Node[] }>();
+  @State() currentActivePosition: number;
 
-  /** list of sorted Stepper items */
-  private items: HTMLCalciteStepperItemElement[] = [];
+  @Watch("currentActivePosition")
+  handlePositionChange(): void {
+    readTask((): void => {
+      this.determineActiveStepper(true);
+    });
+  }
+
+  @State() elWidth: number;
+
+  @Watch("elWidth")
+  handleElWidthChange(): void {
+    readTask((): void => {
+      this.determineActiveStepper();
+    });
+  }
 
-  /** list of enabled Stepper items */
   private enabledItems: HTMLCalciteStepperItemElement[] = [];
 
-  /** keep track of the currently active item position */
-  private currentPosition: number;
+  private itemMap = new Map<HTMLCalciteStepperItemElement, { position: number; content: Node[] }>();
+
+  private items: HTMLCalciteStepperItemElement[] = [];
 
   private mutationObserver = createObserver("mutation", () => this.updateItems());
 
+  /** Specifies if the user is viewing one `stepper-item` at a time when the page width is less than sum of min-width of each item. */
+  private singleViewMode = false;
```

Comment:
Should this be true by default? Assume the smallest size first? I think thats how we decided to handle our breakpoints.

Reply:
make sense, changed this to `multipleViewMode = false`